### ISSUE TITLE
Email address not updating in the UI

### ISSUE THREAD

**Issue by badlard** [praekeltfoundation/vumi-go#721]
Kgosi in Control Room needs to download Guinness VIP stats each week 
GVIP is run in Wambui's account 
Kgosi needed to update Wambui's email address from wambui@praekelt.com to controlroom@praekelt.com 
I changed the email address in go.vumi.org/admin to controlroom@praekelt.com but Kgosi still didn't receive the download in his email. 
I then logged into Wambui's account and changed the email address in the UI 
I received the automated 'account change' email (because I'm an admin) and verified the change by clicking on the link 
Kgosi then received the data download, however the email listed in /account is still wambui@praekelt.com 

In short, the email address didn't update in the UI

**Comment by hodgestar** [comment]
#746 fixed this too.
